### PR TITLE
[12.0][FIX+IMP] website_sale_product_attribute_value_image: Move test tour JS to demo + Add 'alt' and 'title'

### PR DESCRIPTION
**Pull request by Tardo**
Add 'alt' and 'title' html attributes

cc  @tecnativa TT25329

### CODE REVIEW

File: website_sale_product_detail_attribute_value_image/templates/shop_product.xml [hunk 1]
```diff
@@ -28,10 +28,7 @@
                 <t t-foreach="variant_id.value_ids.filtered(lambda x: x.website_product_detail_image_published)" t-as="attr_value">
                     <div class="flex-fill px-1">
                         <div class="text-center">
-                            <span
-                                t-field="attr_value.website_product_detail_image"
-                                t-options="{'widget': 'image', 'resize': '40x40'}"
-                            />
+                            <img class="img" height="40" t-attf-src="/web/image?model=product.attribute.value&amp;field=website_product_detail_image&amp;id=#{attr_value.id}" t-att-title="attr_value.website_name or attr_value.name" t-att-alt="attr_value.website_name or attr_value.name"/>
```
Comment (pedrobaeza) [inline website_sale_product_detail_attribute_value_image/templates/shop_product.xml]: why not max-height and max-width?

Comment (Tardo) [reply website_sale_product_detail_attribute_value_image/templates/shop_product.xml]: You don't want the resize behaviour if the image height is less than 40px?

Comment (pedrobaeza) [reply website_sale_product_detail_attribute_value_image/templates/shop_product.xml]: Resize? The problem I see with this is that an image of 1200x40 will break the layout.

Comment (Tardo) [reply website_sale_product_detail_attribute_value_image/templates/shop_product.xml]: If only sets the "height" or "width" the image preserve the aspect ratio... but yeah, maybe better force the 40x40px to work as before.

Comment (pedrobaeza) [reply website_sale_product_detail_attribute_value_image/templates/shop_product.xml]: That's why I told about max-widht:40 and max-height:40. With that, it adapts to the highest one, and the other is resized in browser preserving aspect ratio.

Comment (Tardo) [reply website_sale_product_detail_attribute_value_image/templates/shop_product.xml:31]: I saw that if the same value is used "max-width" wins... so only need use "max-width" in this case. Thanks!

**Comment by Tardo** [comment]
Changes done

**Review by pedrobaeza — APPROVED** [review]

File: website_sale_product_detail_attribute_value_image/templates/shop_product.xml [hunk 2]
```diff
@@ -28,10 +28,7 @@
                 <t t-foreach="variant_id.value_ids.filtered(lambda x: x.website_product_detail_image_published)" t-as="attr_value">
                     <div class="flex-fill px-1">
                         <div class="text-center">
-                            <span
-                                t-field="attr_value.website_product_detail_image"
-                                t-options="{'widget': 'image', 'resize': '40x40'}"
-                            />
+                            <img class="img" t-attf-src="/web/image?model=product.attribute.value&amp;field=website_product_detail_image&amp;id=#{attr_value.id}" t-att-title="attr_value.website_name or attr_value.name" t-att-alt="attr_value.website_name or attr_value.name"/>
```
Comment (chienandalu) [inline website_sale_product_detail_attribute_value_image/templates/shop_product.xml:32]: Why the website name? The normal would be the attribute name itself as first option, IMO

Comment (Tardo) [reply website_sale_product_detail_attribute_value_image/templates/shop_product.xml:32]: This follows the behavior of 'website_sale_product_attribute_image' which also includes a "website_name" field. Normal is to only use the "name" field, but if you want to change the way they are displayed on the website, you can use "website_name"; in this case, the value of this field wins.

Comment (chienandalu) [reply website_sale_product_detail_attribute_value_image/templates/shop_product.xml:32]: Sorry, you're right. It's ok!

**Review by chienandalu — APPROVED** [review]

**Pull request merged**